### ISSUE TITLE
Expose `isWorker` and `isMaster`

### ISSUE THREAD

**Issue by alexgorbatchev** [strongloop/strong-cluster-control#47]
It would be convenient if you didn't have to `require("cluster")` and could just use one module.

**Comment by sam-github** [comment]
I agree, all public methods and properties of `cluster` would be nice to have on strong-cluster-control, but I don't have the time to implement. I'd be happy to accept a PR.
